docs/src/index.md

```diff
@@ -147,20 +147,127 @@ F-test: 2 models fitted on 50 observations
 ## Methods applied to fitted models
 
 Many of the methods provided by this package have names similar to those in [R](http://www.r-project.org).
+- `adjr2`:  adjusted R² for a linear model (an alias for `adjr²`)
+- `aic`: Akaike's Information Criterion, defined as ``-2 \\log L + 2k``, with ``L`` the likelihood of the model, and `k` it the number of consumed degrees of freedom
+- `aicc`: corrected Akaike's Information Criterion for small sample sizes (Hurvich and Tsai 1989)
+- `bic`: Bayesian Information Criterion, defined as ``-2 \\log L + k \\log n``, with ``L``
+the likelihood of the model, ``k`` is the number of consumed degrees of freedom
 - `coef`: extract the estimates of the coefficients in the model
+- `confint`: compute confidence intervals for coefficients, with confidence level `level` (by default 95%)
+- `cooksdistance`: compute [Cook's distance](https://en.wikipedia.org/wiki/Cook%27s_distance) for each observation in linear model `obj`, giving an estimate of the influence of each data point. Currently only implemented for linear models without weights.
 - `deviance`: measure of the model fit, weighted residual sum of squares for lm's
+- `dispersion`: return the estimated dispersion (or scale) parameter for a model's distribution
+- `dof`: return the number of degrees of freedom consumed in the model, including
+when applicable the intercept and the distribution's dispersion parameter
 - `dof_residual`: degrees of freedom for residuals, when meaningful
+- `fitted`: return the fitted values of the model
 - `glm`: fit a generalized linear model (an alias for `fit(GeneralizedLinearModel, ...)`)
 - `lm`: fit a linear model (an alias for `fit(LinearModel, ...)`)
-- `r2`: R² of a linear model or pseudo-R² of a generalized linear model
-- `stderror`: standard errors of the coefficients
-- `vcov`: estimated variance-covariance matrix of the coefficient estimates
+- `loglikelihood`: return the log-likelihood of the model
+- `modelmatrix`: return the design matrix
+- `nobs`: return the number of rows, or sum of the weights when prior weights are specified
+- `nulldeviance`: return the deviance of the linear model which includs the intercept only
+- `nullloglikelihood`: return the log-likelihood of the null model corresponding to the fitted linear model
 - `predict` : obtain predicted values of the dependent variable from the fitted model
+- `r2`: R² of a linear model (an alias for `r²`)
 - `residuals`: get the vector of residuals from the fitted model
+- `response`: return the model response (a.k.a the dependent variable)
+- `stderror`: standard errors of the coefficients
+- `vcov`: estimated variance-covariance matrix of the coefficient estimates
+
 
 Note that the canonical link for negative binomial regression is `NegativeBinomialLink`, but
 in practice one typically uses `LogLink`.
 
+```jldoctest methods
```

Comment:
This is quite a long example to show on the home page. Also given how simple most of these functions are I'm not sure it's super useful to show all of them. How about adding some of these to the "Linear Model" example section instead?

Reply:
okay

Reply:
Except `coef`, `r2`, `aic` and `prediction`, others are moved to the `Linear Regression` example.

Reply:
I'd rather move everything there to keep the home page simple. Actually we should probably also rework existing examples as it's not super logical to illustrate passing contrasts before even showing how to fit a model... We could move contents to other pages and improve them.

Reply:
Shall we start a different PR for something like `Reorganise GLM documentation` or continue updating this PR only? 
My thought of keeping `r2`, `aic` and `prediction` along with fitting a model at the beginning is, these functionality most of the linear models consumers are looking for.
Looking for your thought.